specification/network/resource-manager/Microsoft.Network/2015-06-15/usage.json

```diff
@@ -131,6 +131,10 @@
             "$ref": "#/definitions/Usage"
           },
           "description": "The list network resource usages."
+        },
+        "nextLink": {
```

Comment:
Example of this property being added exactly like this in a later API Version:
https://github.com/Azure/azure-rest-api-specs/blob/7c6c49a6d8507db1b70909b6b380d7f3bd4168cc/specification/network/resource-manager/Microsoft.Network/2017-06-01/usage.json#L140